sympy/core/sympify.py

```diff
@@ -89,28 +89,27 @@ def _convert_numpy_types(a, **sympify_args):
 
 def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
         evaluate=None):
-    """Converts an arbitrary expression to a type that can be used inside SymPy.
+    """
+    Converts an arbitrary expression to a type that can be used inside SymPy.
+
+    Explanation
+    ===========
 
-    For example, it will convert Python ints into instances of sympy.Integer,
+    It will convert Python ints into instances of sympy.Integer,
     floats into instances of sympy.Float, etc. It is also able to coerce symbolic
     expressions which inherit from Basic. This can be useful in cooperation
     with SAGE.
 
-    It currently accepts as arguments:
-       - any object defined in SymPy
-       - standard numeric python types: int, long, float, Decimal
-       - strings (like "0.09" or "2e-19")
-       - booleans, including ``None`` (will leave ``None`` unchanged)
-       - dict, lists, sets or tuples containing any of the above
-
     .. warning::
-        Note that this function uses ``eval``, and thus shouldn't be used on
-        unsanitized input.
+        Note that this function uses ``eval``, and thus shouldn't be used on sunsanitized input.
```

Comment:
You should keep it as it is. It was dome to make sure the line is less than 79 characters.

Reply:
Can you also split the lines which are 80 characters or more into multiple lines? 
You can check https://www.python.org/dev/peps/pep-0008/#maximum-line-length for reference

Reply:
I see. Will do this.